src/components/Accordion/Accordion.css.ts

```diff
@@ -0,0 +1,71 @@
+import { style } from '@vanilla-extract/css';
+import tokens from '../../tokens';
+import theme from '../../themes/default.theme.css';
+
+export const accordion = style({
+	display: 'flex',
+	flexDirection: 'column',
+});
+
+export const item = style({
+	margin: `${tokens.space.m} 0`,
+	minWidth: '25rem',
+	border: `1px solid ${theme.colors.accordionBorderColor}`,
+	borderRadius: tokens.radii.rectRadius,
+
+	':hover': {
+		borderColor: tokens.colors.gray[200],
```

Comment:
this must come from the theme, from an alias

Reply:
Agreed, I'm only copy-pasting the current styles right now